app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>I am showing</h1>
\ No newline at end of file
+<div>
+    <div>
+        <img src=<%=@user.photo%>>
+        <p>
+            <%=@user.name%>
+        </p>
+        <p>
+            Number of posts: <%=@user.posts_counter%>
+        </p>
+    </div>
+</div>
+<div>
+    <p>Bio</p>
+    <p>
+        <%=@user.bio%>
+    </p>
+</div>
+<div>
+    <% @posts.each do |post| %>
```

Comment:
- [ ] Kindly make sure to use `recent_posts` method to render only the last three posts in the user `show` page